new-games-to-db/start.py

```
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)